README.rst

```diff
@@ -898,6 +898,114 @@ Delete VPN
 
     DELETE /api/v1/controller/vpn/{pk}/
 
+List CA
+^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/pki/ca/
+
+Create CA
+^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/pki/ca/
+
+Import existing CA
+^^^^^^^^^^^^^^^^^^
+
```

Comment:
can you please spend a few words here to explain how to import a CA (specify the fields required to make the import cert work)?